templates/remove-unused-istags.sh.j2

```diff
@@ -36,4 +36,19 @@ echo
 count=$(wc -l <<<${unwanted_imagestream_tags})
 echo "SUMMARY: ${count} ImageStreamTags will be removed"
 
-$OC delete imagestreamtags ${unwanted_imagestream_tags}
+batch_limit=0
+tags_batch=''
+
+for tag in ${unwanted_imagestream_tags}; do
```

Comment:
You can simply use `xargs` to create the batches.
```bash
<<<${unwanted_imagestream_tags} xargs -n ${batch_size} $OC delete imagestreamtags
```

Reply:
fixed